graphite/rules.go

```diff
@@ -109,3 +115,19 @@ func defaultPath(m model.Metric, prefix string) string {
 	}
 	return buffer.String()
 }
+
+func metricLabelsFromPath(path string, prefix string) []*remote.LabelPair {
+	var labels []*remote.LabelPair
+	cleanedPath := strings.TrimPrefix(path, prefix)
```

Comment:
Add a comment explaining that it uses the "default" export format to read back. (and quickly re-explain the format)